.eleventy.js

```diff
@@ -79,6 +79,8 @@ module.exports = function(eleventyConfig) {
     'src/README.md',
     'src/sitemap.xml',
     'src/lib/**/*.js',
+    'src/lib/**/*.html',
+    'src/lib/**/**/*.js',
```

Comment:
I think this is redundant with `'src/lib/**/*.js',` -- the `**` should match any number of subdirectrories.

Reply:
Oh, I see. 
Thanks for pointing out. 
I'll update it.